### PR TITLE
[Fix] Fix metric name

### PR DESCRIPTION
**Pull request by liqikai9**
## Motivation

Remove unneceassary metrix prefix to better set the `save_best`.

## Modification

1. Remove `pck/` prefix in PCKAccuracy-related metrics
2. Remove `nme/` prefix in nme metrics
3. Remove `epe/` prefix in nme metrics
4. Remove `auc/` prefix in nme metrics
5. update related configs and unittests.

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: configs/body_2d_keypoint/topdown_heatmap/jhmdb/td-hm_cpm_8xb32-40e_jhmdb-sub1-368x368.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 auto_scale_lr = dict(base_batch_size=256)
 
 # hooks
-default_hooks = dict(checkpoint=dict(save_best='pck/Mean PCK', rule='greater'))
+default_hooks = dict(checkpoint=dict(save_best='Mean PCK', rule='greater'))
```
Comment (ly015) [inline configs/body_2d_keypoint/topdown_heatmap/jhmdb/td-hm_cpm_8xb32-40e_jhmdb-sub1-368x368.py:30]: Can we just use 'PCK' hear?

File: configs/face_2d_keypoint/topdown_heatmap/300w/td-hm_hrnetv2-w18_8xb64-60e_300w-256x256.py [hunk 2]
```diff
@@ -27,7 +27,7 @@
 auto_scale_lr = dict(base_batch_size=512)
 
 # hooks
-default_hooks = dict(checkpoint=dict(save_best='nme/@[36, 45]', rule='less'))
+default_hooks = dict(checkpoint=dict(save_best='nme', rule='less'))
```
Comment (ly015) [inline configs/face_2d_keypoint/topdown_heatmap/300w/td-hm_hrnetv2-w18_8xb64-60e_300w-256x256.py:30]: Better use uppercase 'NME'

Comment (liqikai9) [reply configs/face_2d_keypoint/topdown_heatmap/300w/td-hm_hrnetv2-w18_8xb64-60e_300w-256x256.py]: Modified.

File: configs/body_2d_keypoint/topdown_heatmap/mpii/td-hm_hourglass52_8xb32-210e_mpii-384x384.py [hunk 3]
```diff
@@ -27,7 +27,7 @@
 auto_scale_lr = dict(base_batch_size=512)
 
 # hooks
-default_hooks = dict(checkpoint=dict(save_best='pck/PCKh', rule='greater'))
+default_hooks = dict(checkpoint=dict(save_best='PCKh', rule='greater'))
```
Comment (liqikai9) [inline configs/body_2d_keypoint/topdown_heatmap/mpii/td-hm_hourglass52_8xb32-210e_mpii-384x384.py:30]: @ly015 How about change the `PCKh` to `PCK` as well?

Comment (ly015) [reply configs/body_2d_keypoint/topdown_heatmap/mpii/td-hm_hourglass52_8xb32-210e_mpii-384x384.py:30]: OK

**Pull request closed without merging**